frontend/src/components/Forms/Fields/index.tsx

```diff
@@ -54,6 +55,29 @@ export const FormInput = ({
     return options ?? defaults;
   };
   switch (fieldType) {
+    case "percentage":
+      return (
```

Comment:
this looks quite similar to the "money" case. I wonder if we'll ever need to add another case?

If so, maybe this is an opportunity to make a re-usable component that wraps a special input component inside a <GridItem><Field></GridItem>?

Reply:
Thought about it, but I left it as it's own for now in case they want something more custom with this type of input.  Good catch though